ui.frontend/src/view/FormContainer.js

```diff
@@ -82,7 +82,11 @@ class FormContainer {
      */
     getParentFormElementId(model) {
         const parentModel = (model.fieldType && model.repeatable) ? model.parent.parent : model.parent;
-        return parentModel.id;
+        if (parentModel.fieldType === 'form') {
```

Comment:
This is required due to @arunattri26's recent change in the last af-core release